packages/core-utils/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "lib/es",
+    "rootDir": "src"
+  },
+  "include": ["src"],
+  "exclude": ["src/**/*.types.ts", "src/**/*.errors.ts"]
```

Comment:
Exclude the files used by `check-dts`. `lodash-fp-types.errors.ts` intentionally includes invalid code and if it were included, the project would not build.